frame/system/src/lib.rs

```diff
@@ -258,6 +255,23 @@ decl_module! {
 			storage::unhashed::put_raw(well_known_keys::CODE, &new);
 		}
 
+		/// Set the new changes trie configuration.
+		pub fn set_changes_trie_config(origin, changes_trie_config: Option<ChangesTrieConfiguration>) {
```

Comment:
OK, too bad that this PR has lived for that long - there were no weights when it was created :( Reading #3157 and seeing that phrase "Since weights are (or better to say: should be) a representation of the time it takes to execute a function", I think if it is better to make this significantly larger than the value for `set_heap_pages`? The reason is that changing trie config leads to skewed digest creation && it takes a lot of time to create any digest changes trie.

OTOH, digest trie could be created at any block, disregarding which transactions have been included in the block. So maybe 10_000 is a good choice. What do you think on that?

Reply:
OK - leaving 20_000 (10_000 for storage change and 10_000 for digest block)